extension/src/vscode/config.ts

```diff
@@ -10,8 +10,11 @@ export enum ConfigKey {
   PYTHON_PATH = 'dvc.pythonPath'
 }
 
-export const getConfigValue = <T = string>(key: ConfigKey): T =>
-  workspace.getConfiguration().get(key, '') as unknown as T
+export const getConfigValue = <T = string, D = string>(
+  key: ConfigKey,
+  defaultValue?: D | T
```

Comment:
does this translates to `defaultValue?: string` ... 😕

Reply:
I think this could be 
```
export const getConfigValue = <T = string>(
  key: ConfigKey,
  defaultValue: T | undefined
): T =>
  workspace.getConfiguration().get(key, defaultValue ?? '') as unknown as T
```